### PR TITLE
docs: add instruction to remove unwanted attributes from systemd logs

### PR DESCRIPTION
**Pull request by sumo-drosiek**
This is feature our customer would like to use. WDYT about adding it to regular configuration as the following?

```
sumologic:
  logs:
    systemd:
      removeAttributes:
      - PRIORITY
```

### Checklist

- [ ] Changelog updated or skip changelog label added
- [ ] Documentation updated
- [ ] Template tests added for new features
- [ ] Integration tests added or modified for major features

### CODE REVIEW

**Review by astencel-sumo — APPROVED** [review]

**Comment by astencel-sumo** [comment]
I agree this should be possible without resorting to `config.merge`. I believe the configuration should allow allowlisting and denylisting attributes.

**Comment by sumo-drosiek** [comment]
> I agree this should be possible without resorting to `config.merge`. I believe the configuration should allow allowlisting and denylisting attributes.

Allowlisting can be little bit complicated AFAIK we have to do it the following way:

1. copy/move allowlisted attributes to some temporary map
2. remove body
3. move temporary map to body

**Comment by swiatekm-sumo** [comment]
Can't we just add `extraProcessors` to the systemd pipeline, same as we have for containers?

File: docs/best-practices.md [hunk 1]
```diff
@@ -353,6 +354,32 @@ In the example above, two internally defined processors were used in metadata pi
...
+              statements:
+                - delete_key(body, "PRIORITY")
+                - delete_key(body, "SYSLOG_FACILITY")
+      service:
+        pipelines:
+          logs/systemd:
+            processors:
+              - logstransform/systemd
+              - transform/cleanup_systemd
+              - batch
```
Comment (sumo-drosiek) [inline docs/best-practices.md:378]:
```suggestion
sumologic:
  logs:
    systemd:
      otelcol:
        extraProcessors:
          - transform/cleanup_systemd:
              log_statements:
                - context: log
                  statements:
                    - delete_key(body, "PRIORITY")
                    - delete_key(body, "SYSLOG_FACILITY")
```

File: docs/best-practices.md [hunk 2]
```diff
@@ -353,6 +354,26 @@ In the example above, two internally defined processors were used in metadata pi
...
+        extraProcessors:
+          - transform/cleanup_systemd:
+              log_statements:
+                - context: log
+                  statements:
+                    - delete_key(body, "PRIORITY")
+                    - delete_key(body, "SYSLOG_FACILITY")
+```
+
+**Note: We do not guarantee that the structure of `otellogs.config.merge` will remain the same between minor helm chart versions**
```
Comment (swiatekm-sumo) [inline docs/best-practices.md:375]:
```suggestion
```

**Review by swiatekm-sumo — APPROVED** [review]

**Pull request merged**